src/node/desktop/src/main/gwt-callback.ts

```diff
@@ -411,10 +422,18 @@ export class GwtCallback extends EventEmitter {
       GwtCallback.unimpl('desktop_close_named_window');
     });
 
-    ipcMain.on(
+    ipcMain.handle(
```

Comment:
As an aside, should we prefer using `handle` to `on` in general, or was there another motivation for this change?

Reply:
I believe `handle` returns a promise. Using `on` requires passing a callback or firing the message with `sendSync`. Either can work to get the main process to respond to the event. I think `handle` is appropriate here to separate the main process doing the actual work from the renderer that is passing along the event from GWT.